storybook/storybook-react/config/preview.tsx

```diff
@@ -11,12 +13,22 @@ export const decorators = [
       <Story />
     </div>
   ),
+  withThemeByClassName({
+    themes: {
+      spacious: 'amsterdam-theme--spacious',
+      compact: 'amsterdam-theme--compact',
+    },
+    defaultTheme: 'spacious',
+  }),
 ]
 
 export const parameters = {
   actions: {
     argTypesRegex: '^on[A-Z].*',
   },
+  backgrounds: {
+    disable: true,
+  },
```

Comment:
Why disable this button?

Reply:
Answer: it makes some of our stories look bad. We’ll restore it when generalizing our approach to show stories on dark backgrounds.